components/CutoutPreviews.tsx

```diff
@@ -26,67 +28,62 @@ const FileMosaicComponent: React.FC<FileMosaicComponentProps> = ({ singleFile, m
   const hasImages = chunks.length > 0 && chunks.every(chunk => chunk.every(file => file.imageUrl));
 
   return (
-    <>
-      <div className="flex items-center justify-center space-x-4">
-        {singleFile ? (
+    <div className={`flex ${isMobile ? 'flex-col' : 'flex-row'} items-center justify-center space-x-4`}>
+      {singleFile ? (
+        <div className={`${isMobile ? 'mb-5' : 'mr-5'}`}>
           <FileMosaic
             key={singleFile.id}
             {...singleFile}
             info
             preview
             uploadStatus={singleFile.uploadStatus}
-            className="mr-5"
             onSee={() => handleSee(singleFile.imageUrl)}
           />
+        </div>
+      ) : (
+        <CssLoader />
+      )}
+      {isMobile ? null : <FaArrowRight className="text-xl" />}
+      {multipleFiles && multipleFiles.length > 0 ? (
+        hasImages ? (
+          <Carousel
+            withIndicators
+            height={200}
+            slideSize={{ base: '100%', sm: '50%', md: '33.333333%' }}
+            slideGap={{ base: 0, sm: 'md' }}
+            loop
+            align="center"
+            orientation={isMobile ? 'vertical' : 'horizontal'}
+          >
```

Comment:
**Suggestion:** Consider using a constant for the `slideSize` and `slideGap` values in the `Carousel` component to avoid hardcoding these values. [best practice]
```suggestion
          const SLIDE_SIZE = { base: '100%', sm: '50%', md: '33.333333%' };
          const SLIDE_GAP = { base: 0, sm: 'md' };
          ...
          <Carousel
            withIndicators
            height={200}
            slideSize={SLIDE_SIZE}
            slideGap={SLIDE_GAP}
            loop
            align="center"
            orientation={isMobile ? 'vertical' : 'horizontal'}
          >
```